oras/container.py

```diff
@@ -54,7 +54,9 @@ def get_blob_url(self, digest: str) -> str:
     def upload_blob_url(self) -> str:
         return f"{self.registry}/v2/{self.api_prefix}/blobs/uploads/"
 
-    def tags_url(self, N=10_000) -> str:
+    def tags_url(self, N=10_000, query=None) -> str:
```

Comment:
Do we still need to accept a query here?

Reply:
na, let's get rid of it! :)